### ISSUE TITLE
billing notification bug

### ISSUE THREAD

**Issue by tayamueller** [ParabolInc/action#1858]
## Issue - Bug

Related to earlier billing notification bug, but NOTE: 
 - this cc is different than previous (@jordanh will need to verify if its old or current)
 - does not clear when I "Clear All Notifications"

![billing bug](https://user-images.githubusercontent.com/14892189/37110654-837f8268-220b-11e8-94ab-352b83366272.jpeg)

- **OS:** macOS
- **Browser:** Chrome
- **Node version:** `node --version`
- **NPM version:** `npm --version`
- **RethinkDB version:** `rethinkdb --version`
- **Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
Users can:
   - Do this
   - Do that
   - Cannot do that

**Comment by ackernaut** [comment]
For clarification, this is one of the notifications that can’t be cleared, by design, until it is addressed.

**Comment by ackernaut** [comment]
Question: was it addressed but still didn’t clear?

**Comment by jordanh** [comment]
I looked at our DB and Stripe today. Our `Organization` record had `stribeSubscriptionId` set to `null` (!). Stripe did not have an active subscription for us (!).

I created a manual one and our March invoice flipped to "Paid". However, the February one will not clear because the subscription no longer exists. Neither will the Notification clear.

Without knowing why our subscription disappeared, we may have to chalk this up to a mystery.

@mattkrick should I delete this notification?

![image](https://user-images.githubusercontent.com/103000/37858898-e61d5642-2ec8-11e8-8a56-578a022170ac.png)

If so, after I do so I will close this for now.

**Comment by mattkrick** [comment]
yeah, whatever we need to do to get to a clean slate is fine. Ideally, we should document that a bug exists post card payment failure. whenever that bug occurs again, its priority should be upgraded to emergency & supersede anything in the current sprint so we can fix it within our app instead of doing a 1-off fix in stripe.

**Comment by jordanh** [comment]
Scrubbed, no change.

**Comment by jordanh** [comment]
Scrubbed, no change
